### DISCUSSION TITLE
widgets packaging functionality

### DISCUSSION THREAD

**Discussion by hellotem** [biolab/orange3#6704] [Ideas]
Imagine you build a big workflow with many widgets spreading over the canvas, it would be nice to have the functionality to package some close related widgets together to have a "super widget", which is displayed as a single new special widget to save canvas space and make the workflow simpler and conciser.

**Comment by janezd** [comment]
This is one of the "futuristic projects" listed in #4090 (coming from #4910). @ales-erjavec has started working on this (https://github.com/biolab/orange-canvas-core/pull/196), but this is an enormous work requiring a lot of redesigning, so I think it's currently stalled.

**Reply by hellotem** [reply]
Thanks for your reply. Another simpler way to compromise is to enable user to draw some basic shapes (like line, rect, circle etc) on the canvas and  color them with different colors, so user could "divide" the canvas into different regions to virtually represent  functionality module (i.e. the data preprocessing region, model implementation region).
